deepfence_server/handler/integration.go

```diff
@@ -44,6 +44,18 @@ func (h *Handler) AddIntegration(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if req.SendSummary {
+		if integration.SupportsSummaryLink(req.IntegrationType) {
+			req.Config["send_summary"] = True
```

Comment:
question: why are we getting send_summary in req and then setting "send_summary" in config, can't we directly take this in config from request?

Reply:
or just keep it in main struct, not inside config would be easier to marshal/unmarshal

Reply:
this config doesn't apply for all integrations and keeping in main struct would required add a new column in the postgresql table